packages/starlight/translations/nl.json

```diff
@@ -0,0 +1,22 @@
+{
+  "skipLink.label": "Naar content",
+  "search.label": "Zoeken",
+  "search.shortcutLabel": "(Druk op / om te zoeken)",
+  "search.cancelLabel": "Annuleren",
+  "search.devWarning": "Zoeken is alleen beschikbaar tijdens productie. \nProbeer om de site te builden en er een preview van te bekijken om lokaal te testen.",
+  "themeSelect.accessibleLabel": "Selecteer thema",
+  "themeSelect.dark": "Donker",
+  "themeSelect.light": "Licht",
+  "themeSelect.auto": "Auto",
+  "languageSelect.accessibleLabel": "Selecteer taal",
+  "menuButton.accessibleLabel": "Menu",
+  "sidebarNav.accessibleLabel": "Main",
```

Comment:
Looks like this wasn’t translated. This I find a bit tricky to translate sometimes. Basically each `<nav>` element on the page is discoverable via shortcuts with assistive technology and so a name distinguishes this one as the “_Main_ Navigation”. In English we just add “Main” because it will usually be read out something like

> “Navigation — Main”

But in some languages that order might be a bit odd, e.g.

> “Navigatie — Hoofd”

Or maybe not? I’m also not 100% sure what a Dutch screenreader would announce.

So maybe “Navigatie — Hoofdnavigatie” is better? Or do you think that just “Hoofd” would work too?

```suggestion
  "sidebarNav.accessibleLabel": "Hoofdnavigatie",
```

Reply:
"Hoofd" on it's own won't work, but something like "Hoofdnavigatie" should be fine!